### ISSUE TITLE
Build of Keras from source fails on Windows "cp: cannot stat 'bazel-bin/keras/tools/pip_package/build_pip_package.runfiles/org_keras/keras': No such file or directory"

### ISSUE THREAD

**Issue by MidouDali** [keras-team/keras#17306]
I used bazel to build Keras from source on Windows 10. I ran this command:
bazel --output_user_root=C:/tmp5 build //keras/tools/pip_package:build_pip_package
Then, when I wanted to create the package with this command:
bazel-bin\keras\tools\pip_package\build_pip_package C:/tmp/keras_pkg
I got this error:
<img width="682" alt="1" src="https://user-images.githubusercontent.com/21183538/205514214-9c0bd639-4a45-4b54-a03f-b91beba67559.png">

It seems that the error came from the buil_pip_pacakge file (line 36):
<img width="706" alt="5" src="https://user-images.githubusercontent.com/21183538/205514266-1923cd32-8a54-4da4-98c3-25c845cf3185.png">

I verified that the folder "bazel-bin" was created:
<img width="667" alt="2" src="https://user-images.githubusercontent.com/21183538/205514236-b34d9a71-82d1-4e01-9c03-96bee4e20b0e.png">

The following figures present the sub-elements of the folders "bazel-bin" and "build_pip_package.exe.runfiles":

<img width="577" alt="3" src="https://user-images.githubusercontent.com/21183538/205514332-37787505-4240-4f90-8538-998e3bfa798a.png">

<img width="655" alt="4" src="https://user-images.githubusercontent.com/21183538/205514473-8e3d0bf9-0d05-4e38-8229-52f7d0bc0f24.png">

**Comment by sushreebarsa** [comment]
@MidouDali To use Keras, you will need to have the TensorFlow package installed. Please check these detailed [instructions](https://www.tensorflow.org/install) and let us know if it helps? This [article](https://towardsdatascience.com/building-keras-from-source-a-follow-along-guide-2bcc4cea3aec) on building Keras from source might help as well. Thank you!

**Comment by MidouDali** [comment]
@sushreebarsa Thank you, I have the Tensorflow package installed. I followed all the steps in the article and got the same error because it seems that Keras requires the creation of the path "bazel-bin/keras/tools/pip_package/build_pip_package.runfiles/org_keras/keras". This path was not created in the "bazel-bin" folder. In addition, I got a folder "build_pip_package.exe.runfiles" and not "build_pip_package.runfiles".
Thank you
